### PR TITLE
chore: automate npm publishing

### PR DESCRIPTION
**Pull request by erezrokah**
This PR automates the `npm publish` part of the release.

### CODE REVIEW

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -8,7 +8,19 @@ jobs:
...
+        if: ${{ steps.release.outputs.release_created }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '15'
+          registry-url: 'https://registry.npmjs.org'
+        if: ${{ steps.release.outputs.release_created }}
+      - run: npm publish
+        if: ${{ steps.release.outputs.release_created }}
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```
Comment (erezrokah) [inline .github/workflows/release-please.yml:26]: I created an added this secret to the repo

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "main": "site/plugins.json",
   "files": [],
   "scripts": {
+    "prepublishOnly": "npm ci && npm run lint && npm run format:ci && npm test",
```
Comment (erezrokah) [inline package.json:8]: We can follow up with another PR to improve our scripts.

File: .github/workflows/release-please.yml (hunk 1 above)
Comment (ehmicky) [reply .github/workflows/release-please.yml:26]: Which user or org's npm token is it?

**Review by ehmicky — APPROVED** [review]

File: .github/workflows/release-please.yml (hunk 1 above)
Comment (erezrokah) [reply .github/workflows/release-please.yml:26]: This new shiny bot account https://www.npmjs.com/~netlify-bot

**Pull request merged**